sdk/servicebus/Azure.Messaging.ServiceBus/README.md

```diff
@@ -58,7 +58,7 @@ To see how to authenticate using Azure.Identity, view this [example](#authentica
 
 ### ASP.NET Core
 
-To inject `ServiceBusClient` as a dependency in an ASP.NET Core app, register the client in the `Startup.ConfigureServices` method:
+To inject `ServiceBusClient` as a dependency in an ASP.NET Core app, first install the package `Microsoft.Extensions.Azure`. Then register the client in the `Startup.ConfigureServices` method:
```

Comment:
I wonder if we should use an "Add the package" approach like we do a few sections above:

https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/servicebus/Azure.Messaging.ServiceBus#install-the-package

Reply:
good idea to be consistent. changed.